pkg/trace/api/api.go

```diff
@@ -101,6 +102,13 @@ func NewHTTPReceiver(
 // Start starts doing the HTTP server and is ready to receive traces
 func (r *HTTPReceiver) Start() {
 	mux := http.NewServeMux()
+
+	mux.HandleFunc("/debug/pprof/", pprof.Index)
+	mux.HandleFunc("/debug/pprof/cmdline", pprof.Cmdline)
+	mux.HandleFunc("/debug/pprof/profile", pprof.Profile)
+	mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
+	mux.HandleFunc("/debug/pprof/trace", pprof.Trace)
```

Comment:
Lets add the one for goroutine as well? `/debug/pprof/goroutine`

Reply:
It's already there as part of the `pprof.Index` handler (along with `heap` & co).